src/ecephys_etl/modules/align_timestamps/__main__.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+
+from allensdk.brain_observatory.argschema_utilities import \
+    ArgSchemaParserPlus, \
+    write_or_print_outputs
+from ._schemas import InputParameters, OutputParameters
+from .barcode_sync_dataset import BarcodeSyncDataset
+from .channel_states import extract_barcodes_from_states, \
+    extract_splits_from_states
+from .probe_synchronizer import ProbeSynchronizer
+
+
+def align_timestamps(args):
```

Comment:
would probably be easier to test/read/doc to split some of these operations into smaller methods